lib/Autocomplete.js

```diff
@@ -10,6 +10,7 @@ let Autocomplete = React.createClass({
     value: React.PropTypes.any,
     onChange: React.PropTypes.func,
     onSelect: React.PropTypes.func,
+    selectOnTab: React.PropTypes.bool.isRequired,
```

Comment:
Best practice is to mark props as `isRequired` if they have a default, as they'll never not be set.